### PR TITLE
Create regions.md

### PR DESCRIPTION
**Pull request by felix-mueller**
## What is the purpose of the change

_Briefly describe the change you are making, this helps the reviewer by providing an overview._

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by felix-mueller** [comment]
@akeller as discussed, happy to get feedback. not urgent

**Comment by akeller** [comment]
@felix-mueller can you add a sentence or something that identifies the default or region for trial accounts? I'm not sure if these are the same from your perspective. For example, could a prospective enterprise customer request a trial in a specific region?

**Review by Sijoma — APPROVED** [review]
LGTM for the regions. 


I agree with @akeller that it could be beneficial to mention which region professional and trial customers get.

**Comment by felix-mueller** [comment]
@Sijoma @akeller added some more notes, wdyt?

File: docs/reference/regions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+In case you are running on a Trial and you would like to try a different region, please reach out to us.
+
+:::note
+
+Interested in any other regions or cloud provider? Please contact us as we are able make additional regions available on request.
+
+:::
+
+## Google Cloud Platform (GCP) regions
```
Comment (akeller) [inline docs/reference/regions.md:23]:
```suggestion
## Available Google Cloud Platform (GCP) regions
```

**Comment by akeller** [comment]
Content wise, this is looking better! Where should we send people when you say to "contact us"? The [Contact Us page](https://camunda.com/contact/) on the website?

Once we've established the info we want on this page, I'd like @christinaausley to review it for structure and clarity.

**Comment by felix-mueller** [comment]
> Where should we send people when you say to "contact us"? The [Contact Us page](https://camunda.com/contact/) on the website?

Yeah that would be fine

**Comment by felix-mueller** [comment]
@akeller when do you think I can see it on the docs page live? :)

**Comment by akeller** [comment]
After @christinaausley reviews, I will do a special release just for you @felix-mueller 😝

File: docs/reference/regions.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+id: regions
+title: "Regions"
+description: "After creating a cluster, specify a region for that cluster. Read on for details of Google Cloud Platform regions currently supported in Camunda Platform 8 SaaS."
+---
+
+When you create a cluster in Camunda Platform 8 SaaS, you must specify a region for that cluster.
+
+Below, find a list of regions currently supported in Camunda Platform 8 SaaS.
+
+Currently, we only make these regions available for customers on the Enterprise Plan. These customers can discuss regions with their Customer Success Manager. Professional and Trial users are currently hosted on US Central.
```
Comment (christinaausley) [inline docs/reference/regions.md:11]: Would it be worth linking to details on the [Enterprise Plan] here for those interested?

**Review by christinaausley — APPROVED** [review]
Technical adjustments in latest commit, including:

- Lengthening description to ~160 characters for SEO purposes.
- Moving a few sentences around for clarity and flow.

Let me know if you have any questions about these changes! One non-blocking comment.

File: docs/reference/regions.md (hunk 2 above)
Comment (akeller) [reply docs/reference/regions.md:11]: Good question, I wasn't able to find a good link on Enterprise Plan details, but I was able to find a separate contact us page. I've asked internally if there is a difference and whether we should include this - https://camunda.slack.com/archives/C01UMP0V17Z/p1672937305910939

**Review by akeller — APPROVED** [review]
I will merge and we can revisit this in another PR if we'd like to add links since it's not entirely clear what link we should use.

**Pull request merged**